### DISCUSSION TITLE
Vercel contribution

### DISCUSSION THREAD

**Discussion by juzarantri** [vercel/community#5789] [Bug Report]
### Question

How can I contribute to vercel documentation error?

**Comment by amyegan** [comment]
Hey @juzarantri! You've come to the right place. Just share the details of what is wrong so I can open a ticket for the error 🙏

**Comment by juzarantri** [comment]
I want to update the documentation on vercel. There is some small typo
error on it. If you can guide me how can I help you it would be great.
